src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.cs

```diff
@@ -11,6 +11,8 @@ namespace System.Security.Cryptography
 {
     public class Rfc2898DeriveBytes : DeriveBytes
     {
+        private static readonly Encoding NoBomUTF8 = new UTF8Encoding(false);
```

Comment:
I realize the name of the field indicates "no bom" and that the previous code wasn't using named parameters, but might be worth using a named parameter here while you're modifying the code.